Subject: mlcustomize: Update virt-customize generated files
From: Richard W.M. Jones rjones@redhat.com Fri Jan 19 13:23:43 2024 +0000
Date: Fri Jan 19 13:24:24 2024 +0000:
Git: 54869c9875318db5e7b7c07be1b17931fbc17895

Modified by this patch to libguestfs:

    generator/customize.ml: Split --chown parameter on ':' character

diff --git a/common/mlcustomize/customize_cmdline.ml b/common/mlcustomize/customize_cmdline.ml
index 245d996..48ee334 100644
--- a/common/mlcustomize/customize_cmdline.ml
+++ b/common/mlcustomize/customize_cmdline.ml
@@ -157,7 +157,7 @@ let rec argspec () =
     let len = String.length arg in
     String.sub arg 0 i, String.sub arg (i+1) (len-(i+1))
   and split_string_triplet option_name arg =
-    match String.nsplit ~max:3 "," arg with
+    match String.nsplit ~max:3 ":" arg with
     | [a; b; c] -> a, b, c
     | _ ->
         error (f_"invalid format for '--%s' parameter, see the man page")
